lib/matplotlib/backends/backend_wx.py

```diff
@@ -409,8 +409,6 @@ class GraphicsContextWx(GraphicsContextBase):
               'miter': wx.JOIN_MITER,
               'round': wx.JOIN_ROUND}
 
-    _dashd_wx = wxc.dashd_wx
```

Comment:
There's a bunch of other entries in wx_compat.py which are not used (e.g. StockCursor) so I think that would belong to a (separate) cleanup of wx_compat.